RELEASE_NOTES.md

```diff
@@ -1,5 +1,152 @@
 # Release Notes
 
+## 2.2.0
+### Features
+- Experimental support for shortname aliasing has been added. This is not enabled by default, but can be turned on by setting the environment variable `CONTAINERS_SHORT_NAME_ALIASING` to `on`.
+- The `podman generate kube` command now features support for exporting container's memory and CPU limits ([#7855](https://github.com/containers/podman/issues/7855)).
+- The `podman play kube` command now features support for setting CPU and Memory limits for containers ([#7742](https://github.com/containers/podman/issues/7742)).
+- The `podman play kube` command now supports Kubernetes configmaps via the `--configmap` flag ([#7567](https://github.com/containers/podman/issues/7567)).
+- The `podman play kube` command now supports a `--log-driver` option to set the log driver for created containers.
+- The `podman run` command now supports the `--network-alias` flag to set network aliases (additional names the container can be accessed at from other containers via DNS if the `dnsname` CNI plugin is in use).
+- The `podman network create` command now supports the `--ipv6` flag to enable dual-stack IPv6 networking for created networks ([#7302](https://github.com/containers/podman/issues/7302)).
+- The `podman inspect` command can now inspect pods, networks, and volumes, in addition to containers and images ([#6757](https://github.com/containers/podman/issues/6757)).
+- The `--mount` option for `podman run` and `podman create` now supports a new type, `image`, to mount the contents of an image into the container at a given location.
+- The Bash and ZSH completions have been completely reworked and have received significant enhancements!
+- The `--log-opt` option for `podman create` and `podman run` now supports the `max-size` option to set the maximum size for a container's logs ([#7434](https://github.com/containers/podman/issues/7434)).
+- The `--network` option to the `podman pod create` create now allows pods to be configured to use `slirp4netns` networking, even when run as root ([#6097](https://github.com/containers/podman/issues/6097)).
+- The `podman pod stop`, `podman pod pause`, `podman pod unpause`, and `podman pod kill` commands now work on multiple containers in parallel and should be significantly faster.
+- The `podman search` command now supports a `--list-tags` option to list all available tags for a single image in a single repository.
+- The `podman search` command can now output JSON using the `--format=json` option.
+- The `podman diff` and `podman mount` commands now work with all containers in the storage library, including those not created by Podman. This allows them to be used with Buildah and CRI-O containers.
+- The `podman container exists` command now features a `--external` flag to check if a container exists not just in Podman, but also in the storage library. This will allow Podman to identify Buildah and CRI-O containers.
```

Comment:
```suggestion
- The `podman container exists` command now features a `--external` option to check if a container exists not just in Podman, but also in the storage library. This will allow Podman to identify Buildah and CRI-O containers.
```